doc/Programming-guide/Key-platform-features/Endpoint-ownership/index.md

```diff
@@ -168,36 +184,41 @@ The configuration should match the following Avro schema.
 }
 ```
 
-### Administration ###
-To create a Twitter owner verifier, use either Administration UI or [Admin REST API]({{root_url}}Programming-guide/Server-REST-APIs/#!/Verifiers/editUserVerifier).
-The following image example illustrates how to create a Twitter owner verifier from Administration UI.
-                                   
+#### Administration
+
+To create a Twitter owner verifier, use  [Administration UI]({{root_url}}Glossary/#administration-ui) or [server REST API]({{root_url}}Programming-guide/Server-REST-APIs/#!/Verifiers/editUserVerifier).
+If you want to use the Administration UI, choose the **Twitter verifier** type from the drop-down list on the **Add user verifier details** page.
+
 ![twitter verifier](Admin-ui/verifier-twitter.png "twitter verifier")
 
-## Custom owner verifier ##
-It is possible to implement and plug-in custom owner verifiers. You can find corresponding instructions on the 
-[Creating custom owner verifier]({{root_url}}Customization-guide/Kaa-Server/Development-environment-setup/#creating-custom-user-verifier) page.
+### Custom owner verifier
 
+You can implement plug in your custom verifiers.
+To do this, follow the instructions in [Creating custom owner verifier]({{root_url}}Customization-guide/Kaa-Server/Development-environment-setup/#creating-custom-user-verifier).
 
-# Endpoint access token flow #
-In the *endpoint access token flow*, new endpoints are attached to the owner with the help of the endpoint which was attached to the owner beforehand.
-The following steps illustrate this flow with the endpoint A, which is already attached to the owner, and the endpoint B, which is due to be attached:
 
-   1. The endpoint B periodically generates and sends its access token to the Kaa cluster.
-   2. The endpoint B displays its access token as a QR code on the screen (TV) or on the webpage (e.g., a router or other device with an embedded server).
-   3. The endpoint A retrieves this token by scanning QR code or in any other suitable way and sends it in the endpoint attach request to the Kaa cluster.
-   4. The Kaa cluster verifies the access code and attaches the endpoint B to the owner of the endpoint A.
+## Using the endpoint access token
```

Comment:
I'd rather give this mode a specific name ("Assisted mode" for example), and then explain that it uses the EP access token